### PR TITLE
Cds 56499 override v2

### PR DESCRIPTION
**Pull request by roshnisarangadharan**
Documented Overrides V2 methods, types, and priority.

### CODE REVIEW

File: docs/continuous-delivery/x-platform-cd-features/overrides-v2.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+Harness has introduced an enhanced experience for service, environment, and infrastructure overrides in Continuous Delivery (CD). A new panel, **Overrides** is added in the **Deployments** section where you can override services, environments, and infrastructures.
+
+Overrides can defined at project, organization, and account levels.
+
+When you create a new project or organization within an account, enable the account resource setting, **Enable Service Override V2** under **Account Settings > Account Resources > Continuous Deployment**.
+
+:::info
+
+Make sure that the `NG_SETTINGS` feature flag is enabled to view this setting.
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/x-platform-cd-features/overrides-v2.md:21]: @thisrohangupta is this ff necessary now? As far as I know, we are removing this FF from code and docs.

File: docs/continuous-delivery/x-platform-cd-features/overrides-v2.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+:::
+
+![overridesV2](./static/overrides-v2.png)
+
+## Limitations
+
+* Runtime inputs are not supported for **Infrastructure Specific** and **Service & Infrastructure Specific** variables.
+
+## Migrating to overrides V2
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/x-platform-cd-features/overrides-v2.md:31]: Will add more info in the upcoming sprint when the UI and APIs are ready.

File: docs/continuous-delivery/x-platform-cd-features/overrides-v2.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+In [Azure App Service configuration](/docs/continuous-delivery/deploy-srv-diff-platforms/azure/azure-web-apps-tutorial/#app-services-configuration), App settings like Application Settings and Connection Strings can be passed as environment variables to the application code. 
+
+You can override Application Settings and Connection Strings from **Global Environment** overrides. 
+
+```mdx-code-block
+  </TabItem>
+</Tabs>
+```
+
+## Override service, environment, and infrastructure settings
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/x-platform-cd-features/overrides-v2.md:89]: Not going in to granular details as the functionality is same for overrides v1 and v2.

**Review by thisrohangupta — DISMISSED** [review]
This looks good

File: docs/continuous-delivery/x-platform-cd-features/overrides-v2.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
+---
+title: Overrides V2
+description: Manage service, environment, and infrastructure overrides.
+sidebar_position: 300
+---
+
+:::info
+
+Currently, this feature beta and is behind the feature flag, `CDS_SERVICE_OVERRIDES_2_0`. Contact [Harness Support](mailto:support@harness.io) to enable the feature.
```
Comment (douglas-j-bothwell) [inline docs/continuous-delivery/x-platform-cd-features/overrides-v2.md:9]:
```suggestion
Currently, this is a beta feature and is behind the feature flag, `CDS_SERVICE_OVERRIDES_2_0`. Contact [Harness Support](mailto:support@harness.io) to enable the feature.
```

File: docs/continuous-delivery/x-platform-cd-features/overrides-v2.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+
+![overridesV2](./static/overrides-v2.png)
+
+## Limitations
+
+* Runtime inputs are not supported for **Infrastructure Specific** and **Service & Infrastructure Specific** variables.
+
+## Migrating to overrides V2
+
+Currently, Harness migrates the existing override configuration to the new version using back end APIs. The existing override scope or scope with all child scopes are migrated based on the requirement.
```
Comment (douglas-j-bothwell) [inline docs/continuous-delivery/x-platform-cd-features/overrides-v2.md:33]:
```suggestion
Currently, Harness migrates the existing override configuration to the new version using back-end APIs. The existing override scope or scope with all child scopes are migrated based on the requirement. 
```

File: docs/continuous-delivery/x-platform-cd-features/overrides-v2.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+
+1. In **Deployments**, select your project, and then select **Overrides**.
+   ![](./static/overrides-v2-1.png)
+2. Select an override method:  
+   - **Global Environment** 
+   - **Service Specific** 
+   - **Infrastructure Specific** 
+   - **Service & Infrastructure Specific** 
+3. Select **New Override**.
+4. Select a project, organization, or account level override.
```
Comment (douglas-j-bothwell) [inline docs/continuous-delivery/x-platform-cd-features/overrides-v2.md:101]:
```suggestion
4. Select a project-, organization-, or account-level override.
```

**Review by douglas-j-bothwell — APPROVED** [review]

File: docs/continuous-delivery/x-platform-cd-features/overrides-v2.md (hunk 6 above)
Comment (roshnisarangadharan) [inline docs/continuous-delivery/x-platform-cd-features/overrides-v2.md:101]:
```suggestion
4. Select a project, organization, or account-level override.
```

**Pull request merged**